testing/sdk_marathon.py

```diff
@@ -132,7 +132,7 @@ def install_app(app_definition: dict) -> (bool, str):
     app_name = app_definition["id"]
 
     with tempfile.TemporaryDirectory() as d:
-        app_def_file = "{}.json".format(app_name)
+        app_def_file = "{}.json".format(app_name.replace('/', '__'))
```

Comment:
See note above on making these changes outside of `dcos-commons`. An alternative would be to create the path to the file.